docs/user/field-selectors.md

```diff
@@ -0,0 +1,149 @@
+| **Boolean**      | `Equals`: Matches if the value is `true` or `false`.<br> `DoubleEquals`: Matches if the value is `true` or `false` (alternative to `Equals`).<br> `NotEquals`: Matches if the value is the opposite of the specified value.<br> `In`: Matches if the value (`true` or `false`) is in the list.<br><i>**Note:** The list can only contain `true` or `false`, so this operator is limited in use.</i><br> `NotIn`: Matches if the value is not in the list.<br> `Exists`: Matches if the field is present.<br> `DoesNotExist`: Matches if the field is not present. | Boolean format (`true`, `false`)                 |
+| **Array**        | `Contains`: Matches if the array contains the specified value.<br> `NotContains`: Matches if the array does not contain the specified value.<br> `In`: Matches if the array overlaps with the specified values.<br> `NotIn`: Matches if the array does not overlap with the specified values.<br> `Exists`: Matches if the field is present.<br> `DoesNotExist`: Matches if the field is not present.<br><br><i>**Note:** Using `Array[Index]` treats the element as the type defined for the array elements (e.g., string, timestamp, number, boolean).</i> | Array element    |
+| **JSON**        | `Equals`: Matches if the value at the specified key path is an exact match to the provided JSON value.<br> `DoubleEquals`: Matches if the value at the specified key path is an exact match to the provided JSON value (alternative to `Equals`).<br> `NotEquals`: Matches if the value at the specified key path does not match the provided JSON value. If the key path does not exist, it is considered a match.<br> `In`: Matches if the value at the specified key path matches at least one JSON value in the provided list.<br> `NotIn`: Matches if the value at the specified key path does not match any of the JSON values in the list. If the key path does not exist, it is considered a match.<br> `Exists`: Matches if the specified key path is present in the JSON field.<br> `DoesNotExist`: Matches if the specified key path is not present in the JSON field.<br> `Contains`: Matches if the value at the specified key path contains the provided JSON value.<br> `NotContains`: Matches if the value at the specified key path does not contain the provided JSON value. | JSON   |
+
+
+## Supported Fields
+
+The following table lists the fields supported for filtering Flight Control objects:
+
+| Field Name                   | Field Type           |
+|------------------------------|----------------------|
+| `metadata.name`              | `String`             |
+| `metadata.owner`             | `String`             |
+| `metadata.labels`            | `StringArray`        |
```

Comment:
Maybe it's worth nothing that `labels` can be queried more extensively using `labelSelector` instead.

Reply:
Added note: 'While `metadata.labels` can be queried using Field Selectors, for more extensive label querying options, consider using Label Selectors.'